.github/actions/setup-node/action.yml

```diff
@@ -16,7 +16,6 @@ inputs:
   node-version:
     description: "Node version to install"
     required: false
-    default: "18"
```

Comment:
A default is no longer needed because we're defaulting to `package.json#engines`.